### ISSUE TITLE
remainder behaves differently between torch and numpy when one input is positive and the other is negative

### ISSUE THREAD

**Issue by tgsmdww** [pytorch/pytorch#49822]
## 🐛 Bug

When remainder one input is positive and the other is negative, the result is different to numpy's result

## To Reproduce

```
import torch
import numpy as np

a = torch.tensor([-57.0625], dtype=torch.half)
b = torch.tensor([1.2339], dtype=torch.half)

print(torch.remainder(a,b))

an = a.numpy()
bn = b.numpy()

print(np.remainder(an, bn))


# output
tensor([0.9375], dtype=torch.float16)  # cpu
[0.953] # numpy
```

## Expected behavior
Expect PyTorch's remainder to match numpy's.

## Environment
```
Collecting environment information...
PyTorch version: 1.5.0+cpu
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Microsoft Windows 10 专业版
GCC version: (x86_64-win32-seh-rev0, Built by MinGW-W64 project) 5.4.0
Clang version: Could not collect
CMake version: Could not collect

Python version: 3.7 (64-bit runtime)
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.18.5
[pip3] numpydoc==1.1.0
[pip3] torch==1.6.0
[pip3] torchvision==0.7.0
[conda] blas                      1.0                         mkl    defaults
[conda] cpuonly                   1.0                           0    pytorch
[conda] mkl                       2020.1                      216    defaults
[conda] mkl-service               2.3.0            py38hb782905_0    defaults
[conda] mkl_fft                   1.1.0            py38h45dec08_0    defaults
[conda] mkl_random                1.1.1            py38h47e9c7a_0    defaults
[conda] numpy                     1.18.5           py38h6530119_0    defaults
[conda] numpy-base                1.18.5           py38hc3f5095_0    defaults
[conda] numpydoc                  1.1.0                      py_0    defaults
[conda] pytorch                   1.6.0               py3.8_cpu_0  [cpuonly]  pytorch
[conda] torchvision               0.7.0                  py38_cpu  [cpuonly]  pytorch
```
## Additional context

cc @mruberry @rgommers @heitorschueroff

**Comment by ngimel** [comment]
Can't repro on master, please update your pytorch version.

**Comment by mruberry** [comment]
I also couldn't reproduce this discrepancy on Colab.

**Comment by gchanan** [comment]
I could reproduce this on 1.5, but it was fixed in 1.6.  Closing as fixed.
